### PR TITLE
Add option to get development build

### PR DESCRIPTION
**Pull request by ludeeus**

### CODE REVIEW

File: get/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,8 @@
+# 1.1.0
+
+- Added option to get development builds
+- Change base images to `*-base-python:3.12-alpine3.20`
+
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply get/CHANGELOG.md:5]: The format need to stay as-is due to rules for the parser.

File: get/Dockerfile [hunk 2]
```diff
@@ -1,8 +1,9 @@
...
-RUN \
-    wget -O /get-hacs https://raw.githubusercontent.com/hacs/get/main/get \
-    && chmod +x /get-hacs
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-CMD ["bash", "/get-hacs"]
+COPY rootfs /
+RUN apk add --no-cache git
+
+CMD [ "/etc/addon/run" ]
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply get/Dockerfile:9]: yes, sure, but that will just cause issues down the line.

File: get/rootfs/etc/addon/run [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    cd /
+    rm -rf /homeassistant/custom_components/hacs
+    mv /tmp/hacs/custom_components/hacs /homeassistant/custom_components/hacs
+    rm -rf /tmp/hacs
+
+    bashio::log.info "Download complete..."
+    bashio::log.info "Remember to restart Home Assistant before you configure it..."
+
+else
+    bashio::exit.nok "Invalid channel!"
```
[the remark this answers is not in the document]
Comment (ludeeus) [reply get/rootfs/etc/addon/run:47]: Its fine, prefered even if these blow up

**Review by frenck — APPROVED** [review]
Nice! That will help with testing a lot! 🤗 

../Frenck

**Comment by ludeeus** [comment]
Thanks for taking a look @frenck 🙏

**Pull request merged**